src/styles/main.scss

```diff
@@ -0,0 +1,131 @@
+@import "products", "cart", "login";
```

Comment:
Please fix the ordering for SASS imports. Some general rules of thumb you can consider are
- import resets
- import global styles
- import overrides (for 3rd party modules you consume)
- import module-level imports like product, cart, login, etc.